packages/terra-worklist-data-grid/tests/wdio/worklist-data-grid-spec.js

```diff
@@ -0,0 +1,109 @@
+const selector = '#default-terra-worklist-data-grid';
+
+Terra.describeViewports('WorklistDataGrid', ['tiny', 'medium', 'large'], () => {
+  describe('with keyboard navigation', () => {
+    beforeEach(() => {
+      browser.url('/raw/tests/cerner-terra-framework-docs/worklist-data-grid/default-worklist-data-grid');
+    });
+
+    afterEach(async () => {
+      await expect(browser.$$('[role="grid"] [tabIndex="0"]')).toBeElementsArrayOfSize(1);
```

Comment:
Why do we need to have this expect statement after every test?

Reply:
Only one cell on the table should have tabIndex=0. If there are multiple cells with tabIndex=0 (e.g if we forgot to remove the tabIndex from the previous cell that has focus), then it would be possible to tab to multiple cells on the table and this would violate the requirement of one tab stop.

Reply:
`before` and `after` functions are intended to be only used for setup and teardown of tests. Not only is it bad practice to put assertions/expects in them but any failures will also prevent execution of subsequent tests. These should be moved into the `it` block of the test itself. Since these are tests, we don't need to worry too much about repetition

Reply:
@sdadn, I understand your point but I do not agree completely. Even tests should be clear and succinct where possible as they too must be maintained, The reason for the setup is to ensure that you know the state of the environment before the test begins. We think that asserting that the actual state is consistent with the expected state can be done in the before/setup. Similarly, teardown is meant to get the state back to some known state so asserting that you have gotten back to that state can be part of the after/teardown.
The actual test then become more succinct as it focuses only on what is actually essential to that test. In this example, the before and after are each one line but consider the case where more elaborate setup/teardown is required. Repeating the code in all the tests would not be desirable.

Reply:
I am only referring to the `expect` function being called in the `afterEach` function. You aren't preforming any operation here, you're only asserting that the state hadn't changed. Assertions are part of the test itself and therefore should be included in the `it` block.

Reply:
In the case of a failure, this also prevents execution of subsequent tests which isn't behavior that we want:

![CleanShot 2023-05-31 at 11 27 01](https://github.com/cerner/terra-framework/assets/38024451/3d1c159e-25f7-466a-9242-297524f5737b)

Reply:
We do want the test to fail for any test where the expectation is violated. I will reach out to you and discuss.

Reply:
After discussion with @Saad and @sycombs, we made the change.